README.rst

```diff
@@ -182,6 +182,9 @@ Usage
       Options to control output and format
 
       -f FMT, --format FMT  set output format [svg, png, ps, pdf, ...]
+      --history-direction {downwards,leftwards,rightwards,upwards}
+                            enforce a specific direction of history on Graphviz
+                            (reverse(!) rankdir, default: "upwards" (rankdir "TB"))
```

Comment:
I am not sure adding `reverse(!) rankdir,` here is beneficial. It only  makes sense if one if familiar with Graphviz vernacular. Perhaps it is better to hide this detail from the user?

Reply:
Can do, I'll follow your intuition.